### PR TITLE
Update build process to get SaaS tests working again

### PR DESCRIPTION
**Pull request by mbwhite**
Using the launch.json fix as a temp change to trigger the PR build

Signed-off-by: Matthew B White <whitemat@uk.ibm.com>

### CODE REVIEW

File: saas-ibp-bootstrap/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# saas-ibp-bootstrap
+
+This directory contains a set of Ansible Playbooks for creating the peer/orderers/channels expected bv the Cucucmber SaaS tests.
```
Comment (jt-nti) [inline saas-ibp-bootstrap/README.md:3]: Minor typo: "bv"

File: saas-ibp-bootstrap/ordering-org-vars.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+api_endpoint: https://xxx
+api_authtype: ibmcloud
+api_key: xxx
+api_secret: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Comment (jt-nti) [inline saas-ibp-bootstrap/ordering-org-vars.yml:8]: These seem to be in several of the vars files. Are they common and if so, could they go in a `credentials-vars.yml` or the `common-vars.yml`?

**Review by jt-nti — APPROVED** [review]
A couple of minor comments but lgtm

File: saas-ibp-bootstrap/ordering-org-vars.yml (hunk 2 above)
Comment (mbwhite) [reply saas-ibp-bootstrap/ordering-org-vars.yml:8]: that is true - these came from the standard tutorial - and in theory each of the orgs and orderers could have different IBP consoles

**Pull request merged**